daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -37,6 +37,11 @@ object SExpr {
 
   sealed abstract class SExprAtomic extends SExpr {
     def evaluate(machine: Machine): SValue
```

Comment:
Once this PR has landed, I suggest we rename this method to something like `lookupValue`. The name `evaluate` is quite confusing when the whole point of atomic expressions is that you can get their value without any further evaluation.

Reply:
Right. I guess the distinction between `evaluate()` and `execute()` is rather subtle, and more explicit terminology would be helpful.

Reply:
renamed to `lookupValue`